packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummaryResult.tsx

```diff
@@ -78,7 +94,7 @@ const WholeMeetingSummaryResult = (props: Props) => {
             </td>
           </tr>
           <tr>
-            <td style={textStyle}>{wholeMeetingSummary}</td>
+            <td style={textStyle} dangerouslySetInnerHTML={{__html: formattedSummary}} />
```

Comment:
-1 We should sanitize the html as marked also [recommends](https://marked.js.org). We already use DOMPurify, so let's use it here as well.